.github/workflows/integration.yml

```
@@ -0,0 +1,18 @@
+name: Integration
+
+on:
+  branches:
```

actionlint (1.7.4)

unknown Webhook event "branches". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)